library/include/ipfs_client/crypto/hasher.h

```diff
@@ -13,7 +13,10 @@ class Hasher {
  public:
   virtual ~Hasher() noexcept {}
 
-  virtual std::optional<std::vector<Byte>> hash(ByteView) = 0;
+  /*! @param bytes Bytes to hash
+   *  @return The digest or nullopt if there was an error
```

Comment:
Doxygen warnings about undocumented items is one of the "maintenance" items I do periodically.